code/game/area/areas/ruins/beachplanet.dm

```diff
@@ -1 +1,16 @@
-//tropical ruins here
+//Ancient Complex
+
+/area/ruin/beach/complex
+	name = "Cybersun Complex"
+	icon_state = "hallC"
+	ambientsounds = MAINTENANCE
+
+/area/ruin/beach/complex/wall
+	name = "Cybersun Complex Exterior Wall"
+	icon_state = "red"
+	ambientsounds = MAINTENANCE
+
+/area/ruin/beach/complex/shuttle
+	name = "Syndicate Shuttle"
+	icon_state = "green"
+	ambientsounds = MAINTENANCE
```

Comment:
you don't need to redefine the ambient sounds for each area, they'll inherit from /beach/complex